python_cross_site_scripting/digitalfabrik_integreat-cms/social_media_headers.py

```
@@ -0,0 +1,371 @@
+    """
+    region = request.region
+    language = region.get_language_or_404(language_slug, only_active=True)
+
+    if not (
+        pn_translation := PushNotificationTranslation.objects.filter(
+            language__slug=language.slug,
+            push_notification__id=slug,
+            push_notification__regions=region,
+        ).first()
+    ):
+        raise Http404("Push Notification not found in this region with this language.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)